modules/apps/27-interchain-accounts/host/keeper/relay_test.go

```diff
@@ -451,57 +452,59 @@ func (suite *KeeperTestSuite) TestOnRecvPacket() {
+				path = NewICAPath(suite.chainA, suite.chainB, encoding)
+				suite.coordinator.SetupConnections(path)
 
-			err := SetupICAPath(path, TestOwnerAddress)
-			suite.Require().NoError(err)
-
-			portID, err := icatypes.NewControllerPortID(TestOwnerAddress)
-			suite.Require().NoError(err)
-
-			// Get the address of the interchain account stored in state during handshake step
-			storedAddr, found := suite.chainB.GetSimApp().ICAHostKeeper.GetInterchainAccountAddress(suite.chainB.GetContext(), ibctesting.FirstConnectionID, portID)
-			suite.Require().True(found)
-
-			icaAddr, err := sdk.AccAddressFromBech32(storedAddr)
-			suite.Require().NoError(err)
-
-			// Check if account is created
-			interchainAccount := suite.chainB.GetSimApp().AccountKeeper.GetAccount(suite.chainB.GetContext(), icaAddr)
-			suite.Require().Equal(interchainAccount.GetAddress().String(), storedAddr)
-
-			suite.fundICAWallet(suite.chainB.GetContext(), path.EndpointA.ChannelConfig.PortID, sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdkmath.NewInt(10000))))
-
-			tc.malleate() // malleate mutates test data
+				err := SetupICAPath(path, TestOwnerAddress, encoding)
+				suite.Require().NoError(err)
 
-			packet := channeltypes.NewPacket(
-				packetData,
-				suite.chainA.SenderAccount.GetSequence(),
-				path.EndpointA.ChannelConfig.PortID,
-				path.EndpointA.ChannelID,
-				path.EndpointB.ChannelConfig.PortID,
-				path.EndpointB.ChannelID,
-				clienttypes.NewHeight(1, 100),
-				0,
-			)
+				portID, err := icatypes.NewControllerPortID(TestOwnerAddress)
+				suite.Require().NoError(err)
 
-			txResponse, err := suite.chainB.GetSimApp().ICAHostKeeper.OnRecvPacket(suite.chainB.GetContext(), packet)
+				// Get the address of the interchain account stored in state during handshake step
+				storedAddr, found := suite.chainB.GetSimApp().ICAHostKeeper.GetInterchainAccountAddress(suite.chainB.GetContext(), ibctesting.FirstConnectionID, portID)
+				suite.Require().True(found)
 
-			if tc.expPass {
+				icaAddr, err := sdk.AccAddressFromBech32(storedAddr)
 				suite.Require().NoError(err)
-				suite.Require().NotNil(txResponse)
-			} else {
-				suite.Require().Error(err)
-				suite.Require().Nil(txResponse)
-			}
-		})
+
+				// Check if account is created
+				interchainAccount := suite.chainB.GetSimApp().AccountKeeper.GetAccount(suite.chainB.GetContext(), icaAddr)
+				suite.Require().Equal(interchainAccount.GetAddress().String(), storedAddr)
+
+				suite.fundICAWallet(suite.chainB.GetContext(), path.EndpointA.ChannelConfig.PortID, sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdkmath.NewInt(10000))))
+
+				tc.malleate(encoding) // malleate mutates test data
+
+				packet := channeltypes.NewPacket(
+					packetData,
+					suite.chainA.SenderAccount.GetSequence(),
+					path.EndpointA.ChannelConfig.PortID,
+					path.EndpointA.ChannelID,
+					path.EndpointB.ChannelConfig.PortID,
+					path.EndpointB.ChannelID,
+					clienttypes.NewHeight(1, 100),
```

Comment:
this was unchanged but we could use `suite.chainB.GetTimeoutHeight()`, which would be equivalent

Reply:
Done